### PR TITLE
Add fn:format-integer with supported pictures: "A", "a", "I" and "i"

### PR DESCRIPTION
**Pull request by rebp**

### CODE REVIEW

**Comment by DrRataplan** [comment]
Awesome!

Can you:

* Run the linter (`npm run lint-fix`)
* Remove the `only`,
* Check whether all other unit tests work?

I'll look further into this tomorrow!

**Comment by rebp** [comment]
Ran the tests and only 17 were pending, the rest did passed

File: src/expressions/functions/builtInFunctions_numeric.ts [hunk 1]
```diff
@@ -42,6 +42,95 @@ const fnAbs: FunctionDefinitionType = (
 	);
 };
 
+const convertIntegerToRoman = (integer: string, isLowerCase?: boolean) => {
+	const romanNumbers = [
```
Comment (DrRataplan) [inline src/expressions/functions/builtInFunctions_numeric.ts:46]: lets pull this to the module scope

File: src/expressions/functions/builtInFunctions_numeric.ts [hunk 2]
```diff
@@ -42,6 +42,95 @@ const fnAbs: FunctionDefinitionType = (
...
+		{ symbol: 'L', decimal: 50 },
+		{ symbol: 'XL', decimal: 40 },
+		{ symbol: 'X', decimal: 10 },
+		{ symbol: 'IX', decimal: 9 },
+		{ symbol: 'V', decimal: 5 },
+		{ symbol: 'IV', decimal: 4 },
+		{ symbol: 'I', decimal: 1 },
+	];
+
+	let int = parseInt(integer);
```
Comment (DrRataplan) [inline src/expressions/functions/builtInFunctions_numeric.ts:62]: Can you make this parseInt(integer, 10)? That fixes the linter error

File: src/expressions/functions/builtInFunctions_numeric.ts [hunk 3]
```diff
@@ -42,6 +42,95 @@ const fnAbs: FunctionDefinitionType = (
...
+
+	if (isLowerCase) {
+		romanString = romanString.toLowerCase();
+	}
+
+	return romanString;
+};
+
+const convertIntegerToAlphabet = (integer: string, isLowerCase?: boolean) => {
+	const ALPHABET = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'.split('');
```
Comment (DrRataplan) [inline src/expressions/functions/builtInFunctions_numeric.ts:77]: Can you pull this to the module lvl?

File: src/expressions/functions/builtInFunctions_numeric.ts [hunk 4]
```diff
@@ -42,6 +42,95 @@ const fnAbs: FunctionDefinitionType = (
...
+
+	let int = parseInt(integer);
```
Comment (DrRataplan) [inline src/expressions/functions/builtInFunctions_numeric.ts:79]: Also add `10` here as a second param

**Review by DrRataplan — CHANGES_REQUESTED** [review]

**Comment by DrRataplan** [comment]
Can you fix the last linter errors?

```/home/runner/work/fontoxpath/fontoxpath/src/expressions/functions/builtInFunctions_numeric.ts:63:2
ERROR: 63:2  semicolon  Unnecessary semicolon
ERROR: 87:2  semicolon  Unnecessary semicolon```

**Review by DrRataplan — APPROVED** [review]

**Pull request merged**